mgmt/ProxyConfig.h

```diff
@@ -98,15 +98,31 @@ class ConfigProcessor
 
 // A Continuation wrapper that calls the static reconfigure() method of the given class.
 template <typename UpdateClass> struct ConfigUpdateContinuation : public Continuation {
+  void (*f)();
   int
   update(int /* etype */, void * /* data */)
   {
     UpdateClass::reconfigure();
     delete this;
     return EVENT_DONE;
   }
+  int
+  update_func(int /*etype*/, void * /* data */)
```

Comment:
If we reload SSLConfigParams then, the certificate update handler will get triggered too, since there are parameters in SSLConfigparams affecting certificate params. Since this patch is intended to update the keyblock only, I don't think we need to reload the whole sslconfigparams.